pandas/core/groupby.py

```diff
@@ -2906,8 +2906,15 @@ def transform(self, func, *args, **kwargs):
                 common_type = np.common_type(np.array(res), result)
                 if common_type != result.dtype:
                     result = result.astype(common_type)
-            except:
-                pass
+            except Exception as exc:
```

Comment:
Sorry, this was a line I put into to see why the change to _find_common_type was causing problems.